source/process.adoc

```diff
@@ -184,7 +184,7 @@ The Specification Document and related technical artifacts must be developed by
 [[efsp-tck]]
 ==== Technology Compatibility Kits
 
-There is exactly one TCK project under an Open Source License for each Specification Version.
+There is exactly one TCK under an Open Source License for each Specification Version.
```

Comment:
Did you decide that the TCK must be developed by an Eclipse project?
If so, should that requirement be included here?

Reply:
I'd rather defer this for now.

Reply:
I've opened #16 to track this.